checkov/terraform/checks/resource/aws/WhoAMI.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
```

Comment:
@TomerSegev241 users can't define owner in the resource type. I think you need a data type check instead.